### PR TITLE
Oop product logging

### PR DESCRIPTION
**Pull request by Daniel-Cretu**

### CODE REVIEW

File: app/Services/ProductionRequestActivityLogger.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+namespace App\Services;
+
+use Carbon\Carbon;
+use Illuminate\Http\Request;
+
+class ProductionRequestActivityLogger extends AbstractRequestActivityLogger
+{
+    protected function collectRequestData(Request $request): array
+    {
+        return ["Url" => $request->fullUrl() ,"Time" => Carbon::now()];
```
Comment (egorbwork) [inline app/Services/ProductionRequestActivityLogger.php]: I propose to include in all your loggers the ip, it provides value in long run.

**Review by egorbwork — APPROVED** [review]
Looks nice, but an improvement can be applied.

File: app/Services/ProductionRequestActivityLogger.php (hunk 1 above)
Comment (Daniel-Cretu) [reply app/Services/ProductionRequestActivityLogger.php]: Added IP to Debug and Production loggers.

**Review by egorbwork — APPROVED** [review]
Now it is fully covering requirements.

**Pull request merged**